batchflow/run_notebook.py

```diff
@@ -0,0 +1,253 @@
+    the executed notebook. Under the hood, we create a separate notebook that saves local variables with names
+    from the `outputs`. After that, we extract output variables in this method and return them.
+    The separate notebook helps to extract outputs in error case.
+
+    Parameters
+    ----------
+    path : str
+        Path to the notebook to execute.
+    inputs : dict, optional
+        Inputs for notebook execution. Converted into a cell of variable assignments and inserted
+        into the notebook on `inputs_pos` place.
+    outputs : str or iterable of str
+        List of notebook local variables that return to output.
+    inputs_pos : int
+        Position to insert the cell with inputs into the notebook.
+    out_path_db : str, optional
+        Path to save the shelve database files. There is no need in files extension.
+        It is a required argument if inputs or outputs are provided.
+    execute_kwargs : dict, optional
+        Other parameters of `:class:ExecutePreprocessor`.
+    out_path_ipynb : str, optional
+        Path to save the output .ipynb file.
+    out_path_html : str, optional
+        Path to save the output .html file.
+    add_timestamp : bool
+        Whether to add a cell with execution information at the beginning of the executed notebook.
+    hide_input : bool
+        Whether to hide the code cells in the executed notebook.
+    display_links : bool
+        Whether to display links to the executed notebook and html at execution.
+    raise_exception : bool
+        Whether to re-raise exceptions from the notebook.
+    return_notebook : bool
+        Whether to return the notebook object from this function.
+
+    Returns
+    -------
+    exec_res : dict
+        Dictionary with the notebook execution results.
+        It provides next information:
+            - 'failed' : bool
+               Whether the execution was failed.
+            - 'outputs' : dict
+               The notebook saved outputs.
+            - 'failed cell number': int
+              An error cell execution number (if exists).
+            - 'traceback': str
+              Traceback message from the notebook (if exists).
+    notebook :class:`nbformat.notebooknode.NotebookNode`
+        Executed notebook object.
+        Note that this output is provided only if `return_notebook` is True.
+    """
+    # pylint: disable=bare-except, lost-exception
+    from IPython.display import display, FileLink
+    import nbformat
+    from jupyter_client.manager import KernelManager
+    from nbconvert.preprocessors import ExecutePreprocessor
+    from nbconvert import HTMLExporter
+    import shelve
+    from dill import Pickler, Unpickler
+    from textwrap import dedent
+
+    if not os.path.exists(path):
+        raise FileNotFoundError(f'Path {path} not found.')
+
+    if (inputs or outputs) and out_path_db is None:
```

Comment:
Unlike other path-parameters, I think that this one should have a default (output_path + postfix)